### PR TITLE
Update Macros section

### PR DESCRIPTION
**Pull request by bravit**
- new structure
- small edits

### CODE REVIEW

File: Macros/Procedural Macros/hello_macro/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+struct Pancakes;
+
+impl HelloMacro for Pancakes {
+    fn hello_macro() {
+        println!("Hello, Macro! My name is Pancakes!");
+    }
+}
+
+fn main() {
```
[the remark this answers is not in the document]
Comment (bravit) [reply Macros/Procedural Macros/hello_macro/src/main.rs:11]: For me, it depends on the open file. If I have `main.rs` open, then it runs normally when I press Run. If I have `lib.rs` open, then I get "Solution isn't runnable" too. I don't know why the plugin behaves like that, but maybe it's ok: the library crate shouldn't be runnable indeed. I tried to add invisible tests, but it seems they are not triggering in theory tasks too, so the behavior is the same. 

So I think we can leave it as it is.

[the remark this answers is not in the document]
Comment (bravit) [reply Macros/Procedural Macros/hello_macro/src/main.rs:11]: I can make lib runnable (or technically buildable) only if there is no `main.rs` in the project.

File: Macros/Procedural Macros/hello_macro_derive/task.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+### Step 2: Defining a Procedural Macro
```
[the remark this answers is not in the document]
Comment (bravit) [reply Macros/Procedural Macros/hello_macro_derive/task.md:1]: I've fixed the wording here.

[the remark this answers is not in the document]
Comment (sofiiako) [reply Macros/Procedural Macros/hello_macro_derive/task.md:1]: thank you!

**Review by sofiiako — APPROVED** [review]

**Pull request merged**